crates/language/src/parsers.rs

```diff
@@ -93,6 +96,7 @@ mod parser_implmentation {
     language_lua,
     language_python,
     language_rust,
+    language_scala,
```

Comment:
Nice work! This will make it easy to add Scala parser support to ast-grep playground!